editor/modes/visual-editor/block-list.js

```diff
@@ -23,14 +24,56 @@ class VisualEditorBlockList extends wp.element.Component {
 		this.onSelectionStart = this.onSelectionStart.bind( this );
 		this.onSelectionChange = this.onSelectionChange.bind( this );
 		this.onSelectionEnd = this.onSelectionEnd.bind( this );
+		this.onPointerMove = throttle( this.onPointerMove.bind( this ), 250 );
```

Comment:
We should `this.onPointerMove.cancel()` in `componentWillUnmount` in case component unmounts while throttled.